compiler/env/OMRClassEnv.hpp

```diff
@@ -83,6 +83,20 @@ class OMR_EXTENSIBLE ClassEnv
    bool isPrimitiveClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazz) { return false; }
    bool isAnonymousClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazz) { return false; }
    bool isValueTypeClass(TR_OpaqueClassBlock *) { return false; }
+
+   /**
+    * \brief
+    *    Checks whether the specified class has fields that are value types that have not been expanded
```

Comment:
expanded seems like the wrong word - this is more like inlined in my opinion - thoughts?

Reply:
Yes, "inlined" better captures the meaning.  Thanks!

Reply:
Revised the Doxygen comments to reflect the new method name, `isZeroInitializable`, so this comment is no longer relevant here.  The Doxygen comments in the downstream pull request in OpenJ9, however, do address this review comment.